src/main/webapp/app/shared/layouts/navbar/navbar.component.html

```diff
@@ -74,7 +74,7 @@ <h6 class="dropdown-header fw-medium" jhiTranslate="global.menu.guidedTutorial">
 <nav class="navbar navbar-dark jh-navbar" [class.expanded]="!isCollapsed">
     <div class="jh-logo-container">
         <a class="navbar-brand" routerLink="/" (click)="collapseNavbar()">
-            <img [src]="SERVER_API_URL + 'public/images/logo.png'" />
+            <img [src]="SERVER_API_URL + 'public/images/logo.png'" height="30" alt="" />
```

Comment:
alt is empty string, as the logo is only decorative. No width set in order to allow for easier change of logos